docker-compose.prod.yml

```diff
@@ -18,11 +18,15 @@ services:
     volumes:
         - static:/static
         - dist:/dist
+    depends_on:
+      - frontend
```

Comment:
Why is nginx depending on the frontend and on web and not the other way around?

Reply:
in theory is not required anymore because the frontend is build during `run`. But, this is here because the "frontend" should be built before start building nginx.